scripts/globals/groundsofvalor.lua

```diff
@@ -403,46 +411,57 @@ function finishGov(player,csid,option,r1,r2,r3,r4,r5,r6,r7,r8,r9,r10,msg_offset)
         player:setVar("fov_numkilled3",0);
         player:setVar("fov_numkilled4",0);
         player:showText(player,msg_cancel);
+        
     elseif (option == GOV_MENU_PAGE_1) then -- Page 1
-        writeRegime(player,r1,msg_accept,msg_jobs,0);
+        writeRegime(player,r1,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_2) then -- Page 2
-        writeRegime(player,r2,msg_accept,msg_jobs,0);
+        writeRegime(player,r2,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_3) then -- Page 3
-        writeRegime(player,r3,msg_accept,msg_jobs,0);
+        writeRegime(player,r3,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_4) then -- Page 4
-        writeRegime(player,r4,msg_accept,msg_jobs,0);
+        writeRegime(player,r4,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_5) then -- Page 5
-        writeRegime(player,r5,msg_accept,msg_jobs,1);
+        writeRegime(player,r5,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_6) then -- Page 6
-        writeRegime(player,r6,msg_accept,msg_jobs,0);
+        writeRegime(player,r6,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_7) then -- Page 7
-        writeRegime(player,r7,msg_accept,msg_jobs,0);
+        writeRegime(player,r7,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_8) then -- Page 8
-        writeRegime(player,r8,msg_accept,msg_jobs,0);
+        writeRegime(player,r8,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_9) then -- Page 9
-        writeRegime(player,r9,msg_accept,msg_jobs,0);
+        writeRegime(player,r9,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_PAGE_10) then -- Page 10
-        writeRegime(player,r10,msg_accept,msg_jobs,1);
+        writeRegime(player,r10,msg_accept,msg_jobs, fov_repeat);
+        
     elseif (option == GOV_MENU_REPEAT_REGIME1) then -- Page 1 Repeat
```

Comment:
you can get rid of all of the REPEAT choices, since they are all no longer possible (and you already put it in fov_repeat)